pkgs/by-name/wp/wprs/package.nix

```diff
@@ -41,6 +50,13 @@ rustPlatform.buildRustPackage rec {
 
   preFixup = ''
     cp  wprs "$out/bin/wprs"
+
+    # wprs is a raw python script that needs a module.
+    wrapPythonProgramsIn "$out/bin" "${pythonPath}"
```

Comment:
Did you try without this? In theory the shebang will be rewritten to python with the package

Reply:
It works without `wrapPythonProgramsIn "$out/bin" "${pythonPath}"`.
it seems like `python3.pkgs.wrapPython` is not needed either?
I built it again with just adding the python3.withPackages line to buildInputs and it doesn't give the error.

Reply:
Yes, I would assume that.

Reply:
> Did you try without this? In theory the shebang will be rewritten to python with the package

TIL. That works. Thanks!